demisto_sdk/commands/lint/tests/test_pylint_plugin/base_checker_test.py

```diff
@@ -616,6 +674,12 @@ def test_commands_dismiss_for_feeds_checker(self):
         assert node_a
         with self.assertNoMessages():
             self.checker.visit_importfrom(node_a)
+        with self.assertAddsMessages(
+                pylint.testutils.Message(
+                    msg_id='unimplemented-test-module',
+                    node=node_a
+                )
+        ):
             self.checker.leave_module(node_a)
```

Comment:
I suggest adding another small test in which the code includes the `test-module` in the dict and making sure that there is no warning message regarding the test module implementation .( As in all the other cases we suppose that the message should appear.)